tekton/images/test-runner/Dockerfile

```diff
@@ -54,8 +57,9 @@ RUN apt update && apt install -y --no-install-recommends \
     zlib1g-dev \
     && rm -rf /var/lib/apt/lists/* \
     && update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
-    && update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1 \
-    && python -m pip install --upgrade pip setuptools wheel
+    && update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
+    # \
+    # && python -m pip install --upgrade pip setuptools wheel
```

Comment:
Why this is here as a comment? This is covered now by `python3-setuptools` right?

Reply:
Yes, my bad, I forgot to remove that comment 😓